app/views/shared/_user.html.erb

```diff
@@ -0,0 +1,7 @@
+<li class="user">
+  <%= image_tag user.photo %>
+  <div class="blue-box">
+    <h2><%= link_to user.name, user_path(user) %></h2>
+    <p>Number of posts: <%= if user.posts.any? then user.postCounter else 0 end %></p> 
```

Comment:
The number of posts is not being displayed correctly, I created multiple posts for this user, and yet 0 is being displayed: 
![image](https://github.com/ezraermy/rails-blog/assets/93520178/b4163632-a83e-4576-8eda-b6805478f632)